### PR TITLE
feat: enable regular withdrawals for privy users [OTE-282]

### PR DESCRIPTION
**Pull request by yogurtandjam**
withdrawals non-privy wallet
Screenshot 2024-04-30 at 2 40 38 PM

deposits non-privy wallet
Screenshot 2024-04-30 at 2 40 19 PM

withdraw with privy wallet
Screenshot 2024-04-29 at 4 21 53 PM

deposits with privy wallet
Screenshot 2024-04-30 at 1 59 25 PM

### CODE REVIEW

**Review by rosepuppy — CHANGES_REQUESTED** [review]
I was wrong about deleting code. We still need to gate the chain options for deposit, but allow the options it for withdrawals. So just need to add the check for only when `type === TransferType.deposit`

**Comment by yogurtandjam** [comment]
> I was wrong about deleting code. We still need to gate the chain options for deposit, but allow the options it for withdrawals. So just need to add the check for only when `type === TransferType.deposit`

oh right. that's my bad. i keep forgetting this component is shared between deposits and withdrawals. change will be made

File: src/views/forms/AccountManagementForms/SourceSelectMenu.tsx [hunk 1]
```diff
@@ -92,7 +90,8 @@ export const SourceSelectMenu = ({
           groupLabel: stringGetter({ key: STRING_KEYS.EXCHANGES }),
           items: exchangeItems,
         },
-        walletType !== WalletType.Privy &&
+        // only unblock privy wallets for withdrawals
+        type === TransferType.withdrawal &&
```
Comment (yogurtandjam) [inline src/views/forms/AccountManagementForms/SourceSelectMenu.tsx:94]: @rosepuppy this is what you were talking about, right?

Comment (rosepuppy) [reply src/views/forms/AccountManagementForms/SourceSelectMenu.tsx:94]: actually so it should only block if it is deposit and privy, so something like:
`type === TransferType.deposit && walletType !== WalletType.Privy`

Comment (yogurtandjam) [reply src/views/forms/AccountManagementForms/SourceSelectMenu.tsx:94]: omg 🤦 sorry i gotta slow down when making these. yeah this should have been really clear, my bad

**Review by rosepuppy — APPROVED** [review]

**Pull request merged**